.github/workflows/fetch-and-ingest-genbank-master.yml

```diff
@@ -60,11 +60,11 @@ jobs:
         name: Set config overrides
         env:
           TRIAL_NAME: ${{ inputs.trial_name }}
-          FETCH_FROM_DATABASE: ${{ inputs.fetch_from_database != '' && inputs.fetch_from_database || true }}
+          FETCH_FROM_DATABASE: ${{ github.event_name != 'workflow_dispatch' && true || inputs.fetch_from_database }}
         run: |
           config="--config"
 
-          if [[ "$FETCH_FROM_DATABASE" ]]; then
+          if "$FETCH_FROM_DATABASE"; then
```

Comment:
This relies on running the programs `true` and `false`... 🫣

Reply:
Thanks for flagging! Updating in https://github.com/nextstrain/ncov-ingest/pull/444